tests/Form/FormMapperTest.php

```diff
@@ -235,6 +235,56 @@ public function testWithFieldsCascadeTranslationDomain(): void
         ]], $this->admin->getFormGroups());
     }
 
+    public function testWithFieldsCascadeTranslationDomainFalse(): void
+    {
+        $this->contractor->expects($this->once())
+            ->method('getDefaultOptions')
+            ->willReturn([]);
+
+        $this->formMapper->with('foobar', [
+            'translation_domain' => false,
+        ])
+            ->add('foo', 'bar')
```

Comment:
Since it's a `FormMapper` I think we should use proper values like `TextType::class` (as type), if we specify `class-string` as type, this would fail.

Reply:
`'collection'` is allowed for instance, so `$type` is not always a class-string.

Reply:
Passing `collection` is clearly wrong and should be deprecated.